src/main/java/frc/robot/commands/vision/ShiftLeft.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.vision;
+import frc.robot.Constants;
+import frc.robot.subsystems.Vision;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class ShiftLeft extends CommandBase {
+  /** Creates a new ShiftLeft. */
+  private final Vision vision;
+  public ShiftLeft(Vision vision) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    this.vision = vision;
+    addRequirements(this.vision);
```

Comment:
I think you can converge down to a single command for each the right and left shift. These would be more so drive commands IMO. The command could access both the vision and drive if needed but only require the drive subsys like we discussed at end of Thursday